src/components/Sidebar.tsx

```diff
@@ -245,6 +263,31 @@ const Sidebar = ({ onClose, ...rest }: SidebarProps) => {
 
 export default Sidebar;
 
+export function ManageMenu({ user } : {
```

Comment:
No need for `export`. Rename to `ManagerDropdownMenu`. No need to pass in `user`. Call useUserContext within the function.